telegram_menu/navigation.py

```diff
@@ -298,7 +298,6 @@ async def goto_menu(
         if message is None:
             return -1  # message was not sent, abort
         menu_message.is_alive()
-        menu_message.message_id = message.message_id
```

Comment:
Can I ask you why this line is wrong? Without it,  non inlined BaseMessages will get message_id always equal to -1 and so there will be no way of exiting bot software (deleting StartMessage will be impossible).

Reply:
Ok I see your point, it's fixed